### ISSUE TITLE
weird NPE in optionsoverlay

### ISSUE THREAD

**Issue by yugecin** [itdelatrisu/opsu#246]
I changed the skin in the optionmenu, and when I pressed the restart button it crashed.
Running the latest code (at 25356ce)
Cannot reproduce the crash atm. I do remember I entered 'skin' to search in the optionsmenu.
The stacktrace tells that some dropdown was handling a click but none was open at that time.

The crash popup also crashed when I wanted to report it, so here is the log contents:
```
Thu Feb 02 23:36:11 CET 2017 ERROR:null
java.lang.NullPointerException
	at itdelatrisu.opsu.options.OptionsOverlay$1.itemSelected(OptionsOverlay.java:323)
	at itdelatrisu.opsu.ui.DropdownMenu.mousePressed(DropdownMenu.java:357)
	at org.newdawn.slick.Input.poll(Input.java:1241)
	at org.newdawn.slick.GameContainer.updateAndRender(GameContainer.java:680)
	at itdelatrisu.opsu.Container.gameLoop(Container.java:110)
	at itdelatrisu.opsu.Container.start(Container.java:76)
	at itdelatrisu.opsu.Opsu.main(Opsu.java:216)
Thu Feb 02 23:36:30 CET 2017 ERROR:An error occurred in the crash popup.
Thu Feb 02 23:36:30 CET 2017 ERROR:null
java.lang.NullPointerException
	at java.net.URLEncoder.encode(URLEncoder.java:204)
	at itdelatrisu.opsu.ErrorHandler.getIssueURI(ErrorHandler.java:180)
	at itdelatrisu.opsu.ErrorHandler.getIssueURI(ErrorHandler.java:262)
	at itdelatrisu.opsu.ErrorHandler.error(ErrorHandler.java:145)
	at itdelatrisu.opsu.Container.start(Container.java:92)
	at itdelatrisu.opsu.Opsu.main(Opsu.java:216)
```

**Comment by itdelatrisu** [comment]
Yeah, noticed this too.  Looking into it now...

**Comment by itdelatrisu** [comment]
Should be fixed in 5d991dc -- forgot that `itemSelected()` only fires if the option changes, so we don't need the extra checks.  Let me know if something's still wrong.

Fixed `ErrorHandler` crashing in db376af, didn't know that `e.getMessage()` could be null...
